packages/token-vault/src/lib/client.ts

```diff
@@ -88,18 +116,35 @@ export function client(config: ClientConfig) {
         };
       });
     },
+
+    /** ****************************************************
+     * @method store - Register the Token Vault Store
+     * @returns {Promise<Tokens>} - The Token Vault token store object
+     * @example const tokenStore = register.store();
+     */
     store: function () {
       const clientId = config?.forgerock?.clientId || 'WebOAuthClient';
-      const hasTokenEventName = config?.events?.has || 'HAS_TOKENS';
-      const refreshTokenEventName = config?.events?.refresh || 'REFRESH_TOKENS';
-      const removeTokenEventName = config?.events?.remove || 'REMOVE_TOKENS';
+      const hasTokenEventName = config?.events?.has || 'TVP_HAS_TOKENS';
+      const refreshTokenEventName =
+        config?.events?.refresh || 'TVP_REFRESH_TOKENS';
+      const removeTokenEventName =
+        config?.events?.remove || 'TVP_REMOVE_TOKENS';
 
       return {
-        get() {
+        /**
+         * @method get - A noop method that replaces the default get method
+         * @returns {Promise<null>}
+         */
+        get(clientId: string): Promise<Tokens> {
           // Tokens are not retrievable from the iframe
           return Promise.resolve(null as unknown as Tokens);
         },
-        has() {
+
+        /**
+         * @method has - Check if tokens exist in the Token Vault
+         * @returns {Promise<{ hasTokens: boolean }>} - True if tokens exist, false if not
+         */
+        has(): Promise<{ hasTokens: boolean }> {
           const proxyChannel = new MessageChannel();
```

Comment:
There could be a performance consideration with creating a message channel for each of these calls. Mainly if the calls are done in a long loop and these message channels are rapidly generated faster than they can be cleaned up. It may take the Garbage Collector a while to clean them up.

The reason for doing this, though, is to ensure reference identity for the request-response pairing. Creating a single message channel and trying to reuse it, you could have race conditions resulting in responses that are mismatched with requests.